src/unittest/test_keycode.cpp

```diff
@@ -15,20 +15,26 @@ class TestKeycode : public TestBase {
 
 	void runTests(IGameDef *gamedef);
 
+	/* TODO: Re-introduce unittests after fully switching to SDL.
```

Comment:
SDL-specific unittests would be nice, but those can be added in another PR too.

Reply:
I would prefer this in a separate PR. IMO this can wait a bit, possibly until after we refactor the keypress system to use SDL directly.